core/DataAccess/LogQueryBuilder/JoinTables.php

```diff
@@ -137,5 +132,147 @@ private function checkTableCanBeUsedForSegmentation($tableName)
         }
     }
 
-    
+    private function parseDependencies(array $tables)
+    {
+        // NOTE: joins can be specified explicitly as arrays w/ 'joinOn' keys or implicitly as table names. when
+        // table names are used, the joins dependencies are assumed based on how we want to order those joins.
+        // the below table list the possible dependencies of each table, and is specifically designed to enforce
+        // the following order:
+        // log_link_visit_action, log_action, log_visit, log_conversion, log_conversion_item
+        // which means if an array is supplied where log_visit comes before log_link_visitAction, it will
+        // be moved to after it.
+        static $implicitTableDependencies = [
```

Comment:
is that needed to be static?

Reply:
Not really, it's just a constant. I can remove the static modifier.